stdlib/public/core/AtomicInt.swift.gyb

```diff
@@ -65,7 +65,7 @@ internal func _swift_stdlib_atomicCompareExchangeStrongInt(
   object target: UnsafeMutablePointer<Int>,
   expected: UnsafeMutablePointer<Int>,
   desired: Int) -> Bool {
-#if arch(i386) || arch(arm)
+#if arch(i386) || arch(arm) || arch(wasm32)
```

Comment:
Other comments that you left say that WebAssembly has no atomics right now. So how do these functions get compiled? Regular loads and stores?

Reply:
Yes, this is set in `IRGen.cpp` change in this same PR:

```swift
  if (Clang->getTargetInfo().getTriple().isOSBinFormatWasm())
    TargetOpts.ThreadModel = llvm::ThreadModel::Single;
```